_posts/2024-07-15-get-together-ome-transforms.markdown

```diff
@@ -0,0 +1,48 @@
+
+This hackathon will focus on **advancing [OME-Zarr] spatial transformations**.
+
+## Overview of OME-Zarr
+
+[OME-Zarr] is a cloud-optimized bioimaging file format that enjoys international community support and widespread adoption in neuroscience. It supports large-scale bioimages with spatial metadata, making it a critical tool for scientific research. The current OME-Zarr standard is enhanced by the [coordinate transformations draft], which introduces robust support for spatial transformations. This is particularly important for neuroimaging and other scientific imaging practices, as it facilitates:
+
+- **Reproducibility and Consistency:** Explicit support for spatial transformations ensures consistent application across various platforms and applications. This feature aligns with the [FAIR] principles, enabling independent researchers to verify results.
+- **Integration with Analysis Workflows:** By treating spatial transformations as a first-class entity within file formats, OME-Zarr allows seamless integration with diverse image analysis workflows, eliminating the need for additional conversion steps.
+- **Efficiency and Accuracy:** Embedding transformations within the file format minimizes the need for re-sampling, thereby reducing sampling errors and preserving analysis accuracy. This standardization is crucial for handling the massive data volumes generated by modern microscopy techniques.
+- **Flexibility in Analysis:** Native support for spatial transformations provides researchers with the flexibility to apply, modify, or reverse transformations as needed, facilitating longitudinal studies, multi-modal imaging, and comparative analysis.
+
+## Hackathon Agenda
+
+The hackathon is structured into three key components:
+
+1. **Day 1:** Tutorial sessions covering the application needs for coordinate transformations, mathematical principles, and current computational standards and tools available in the open-source ecosystem.
+2. **Day 2:** Small working groups will review and propose enhancements to the current coordinate transformations draft and relevant neuroimaging additions.
+3. **Day 3:** Hands-on activities where participants will implement and apply the proposed improvements to the standards.
+
+## Event Details
+
+- **Dates:** Friday, July 26th - Sunday, July 28th, 2024
+- **Location:** Hybrid event at the University of North Carolina-Chapel Hill, via Google Meet videoconferencing, Image.sc Island Gather.Town virtual space, and Image.sc Zulip Chat.
+- **Cost:** [Registration] is free!
+
+**[Register now and add the event to your calendar!](https://insightsoftwareconsortium.github.io/GetYourBrainTogether/HCK03_2024_UNC_Hybrid/)**
+
+[Registration]: https://forms.gle/LL4quQsbSWawKYSa6
```

Comment:
is the duplication of the registration link intended?

Reply:
I think it's intended in the sense that it let's you use "`[Register]`" and "`[Registration]`" in the text without the longer-form of "`[Register][Registration]`".